src/window.py

```diff
@@ -85,6 +85,10 @@ class PortfolioWindow(Handy.ApplicationWindow):
     deck = Gtk.Template.Child()
     headerbar = Gtk.Template.Child()
     placeholder_box = Gtk.Template.Child()
+    error_box = Gtk.Template.Child()
+    error_label = Gtk.Template.Child()
+    error_description = Gtk.Template.Child()
+    error_button = Gtk.Template.Child()
```

Comment:
Instead of adding new UI components, I would suggest:

1. Add a `failed` signal to https://github.com/tchx84/Portfolio/blob/master/src/worker.py#L150
2. Connect to it https://github.com/tchx84/Portfolio/blob/master/src/window.py#L251
3. Emit the `failed` signal here https://github.com/tchx84/Portfolio/blob/master/src/worker.py#L165 (or try check permissions in the worker)
4. Display the existing error screen as it is done for the delete operation here https://github.com/tchx84/Portfolio/blob/master/src/window.py#L762